cloudconfig/machinecloudconfig.go

```diff
@@ -81,7 +81,7 @@ func NewMachineInitReaderFromConfig(cfg MachineInitReaderConfig) InitReader {
 // machine. It is sourced from both Cloud-Init and Curtin data.
 func (r *MachineInitReader) GetInitConfig() (map[string]interface{}, error) {
 	switch utilsos.OSTypeForName(r.config.Base.OS) {
-	case utilsos.Ubuntu, utilsos.CentOS, utilsos.OpenSUSE:
+	case utilsos.Ubuntu, utilsos.CentOS:
```

Comment:
Unsure if we are removing Centos entirely as 7 still seems to be supported but should we remove it here if it's a full rip out?

Reply:
We are keeping CentOS 7 so we need to keep this here